test/spec/modules/consentManagement_spec.js

```diff
@@ -667,6 +674,59 @@ describe('consentManagement', function () {
           expect(consent.apiVersion).to.equal(2);
         });
 
+        it('produces gdpr metadata', function () {
+          let testConsentData = {
+            tcString: 'abc12345234',
+            gdprApplies: true,
+            purposeOneTreatment: false,
+            eventStatus: 'tcloaded',
+            vendorData: {
+              tcString: 'abc12345234'
+            }
+          };
+          cmpStub = sinon.stub(window, '__tcfapi').callsFake((...args) => {
+            args[2](testConsentData, true);
+          });
+
+          setConsentConfig(goodConfigWithAllowAuction);
+
+          requestBidsHook(() => {
+            didHookReturn = true;
+          }, {});
+          let consentMeta = gdprDataHandler.getConsentMeta();
+          sinon.assert.notCalled(utils.logError);
+          expect(consentMeta.consentStringSize).to.be.above(0)
+          expect(consentMeta.gdprApplies).to.be.true;
+          expect(consentMeta.apiVersion).to.equal(2);
+          expect(consentMeta.timestamp).to.be.above(1644367751709);
+        });
+
+        it('performs lookup check and stores consentData for a valid existing user with additional consent', function () {
```

Comment:
This looks like a copy of the test case below it?

Reply:
The setup is the same, but the assertions related to the getConsentMeta() return:

```
          let consentMeta = gdprDataHandler.getConsentMeta();
          sinon.assert.notCalled(utils.logError);
          expect(consentMeta.consentStringSize).to.be.above(0)
          expect(consentMeta.gdprApplies).to.be.true;
          expect(consentMeta.apiVersion).to.equal(2);
          expect(consentMeta.timestamp).to.be.above(1644367751709);
```

vs. 

```
          let consent = gdprDataHandler.getConsentData();
          sinon.assert.notCalled(utils.logError);
          expect(didHookReturn).to.be.true;
          expect(consent.consentString).to.equal(testConsentData.tcString);
          expect(consent.addtlConsent).to.equal(testConsentData.addtlConsent);
          expect(consent.gdprApplies).to.be.true;
          expect(consent.apiVersion).to.equal(2);
```

The checks for `consentStringSize` and `timestamp` are specifically novel.

Reply:
I may be having a stroke, but I don't see any call to `getConsentMeta` (test case in lines 704-729). `diff` tells me it's identical to the test case from lines 730-754.

Reply:
Nope, looks like I had the stroke. You're right, that's an unnecessary copy/paste. Let me pull the 704 block out and commit it. Thanks!